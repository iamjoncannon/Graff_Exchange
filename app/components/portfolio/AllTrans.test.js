


describe("AllTrans", ()=>{

    it('1) renders successfully with data from redux store', ()=>{

        // let tree = component.toJSON();
        
        // expect(tree).toMatchSnapshot();
    })

    it("2) renders five title spans", ()=>{

        // const className = "indiv-trans indiv-container"

        // let parent = component.root.find( node=> node.props.className === className)

        // let spans = parent.children[0].props.children.map(child => child.props.children)
        
        // expect(spans).toEqual( ['Type', 'Quantity', 'Price', 'Date' ])
    
    })

    it("3) renders transaction history item in five spans", ()=>{

        // const className = "indiv-trans indiv-container"

        // let parent = component.root.find( node=> node.props.className === className)

        // let spans = parent.children[1].props.children[0].props.children.map(child => child.props.children)

        // expect(spans).toEqual([ 'Buy', 100, [ '$', 10.329999923706055 ], 'Wed Jul 17 2019 ' ])

    })

})