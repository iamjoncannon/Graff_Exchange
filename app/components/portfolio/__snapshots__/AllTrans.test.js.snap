// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllTrans 1) renders successfully with data from redux store 1`] = `
<div
  className="all-trans"
>
  <span>
    Transaction History
  </span>
  <div>
    <span>
      Type
    </span>
    <span>
      Symbol
    </span>
    <span>
      Quantity
    </span>
    <span>
      @Buy
    </span>
    <span>
      Date
    </span>
  </div>
  <div>
    <div
      className="_allTransItem"
    >
      <span>
        Buy
      </span>
      <span>
        F
      </span>
      <span>
        100
      </span>
      <span>
        $
        10.33
      </span>
      <span>
        Wed Jul 17 2019 
      </span>
    </div>
  </div>
</div>
`;
